tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -118,15 +118,10 @@ def run_pipeline_test(self, speech_recognizer, examples):
                 },
             )
         else:
-            # Non CTC models cannot use chunk_length
-            with self.assertRaises(ValueError) as v:
-                outputs = speech_recognizer(audio, chunk_length_s=10)
-            self.assertEqual(v.exception, "")
-
             # Non CTC models cannot use return_timestamps
             with self.assertRaises(ValueError) as v:
                 outputs = speech_recognizer(audio, return_timestamps="char")
-            self.assertEqual(v.exception, "")
+            self.assertEqual(str(v.exception), "We cannot return_timestamps yet on non-ctc models !")
```

Comment:
Why not use `self.assertRaisesRegex` here to test the exception and the message together?

Reply:
Can do that - I just followed what has been there.

Reply:
Done!